src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -248,7 +248,7 @@ export default function RequestedAppointmentDetailsPage() {
           You shared these details about your concern
         </h2>
         {!isCCRequest && apptDetails}
-        {isCCRequest && <>{comment || 'none'}</>}
+        {isCCRequest && <>{apptDetails || 'none'}</>}
```

Comment:
I think this was just `comment` before because we don't ask the reason for visit questions in the CC flow.

Reply:
Gotcha. Switching back to `comment`.